### ISSUE TITLE
Editor does not show up the dynamic code (BUG)

### ISSUE THREAD

**Issue by sjcantony** [FormidableLabs/react-live#281]
Live Editor does not show up the dynamic code.  it shows blank for the first time, even after the code is assigned a value by a variable.  when switching between pages and come back to live editor, it shows the code. ***(BUG)***

**Comment by daniele-zurico** [comment]
same problem is present on my end:
```
<LiveProvider code={state.codeToRender} scope={scope}>
            <LivePreview />
            <LiveEditor className="liveEditor" />
            <LiveError className="liveError" />
          </LiveProvider>
```

LivePreview render the component
LiveEditor has the div code populated but the text area is empty

**Comment by elchininet** [comment]
Same issue here, it doesn‘t happen in version `2.2.3`.

**Comment by jpdriver** [comment]
please can someone provide either a CodeSandbox or a sample repo so I can take a look? i'm not aware of any issues and haven't seen anything like this yet..

**Comment by stevenocchipinti** [comment]
Here is a [quick re-production codesandbox](https://codesandbox.io/s/react-live-bug-repro-7y7rm?file=/src/index.js), looks like it the issue only started happening from `v2.3.0`.

If you press the button to change the code, the preview updates but the code does not.
If you change the version to something earlier than `2.3.0`, both the code and the preview will update.

**Comment by stevenocchipinti** [comment]
Awesome, thank you! 😄 
Looking forward to the release so I can remove my patch! 🎉

**Comment by stevenocchipinti** [comment]
@jpdriver, not to hassle you about it but do have a rough ETA for the release for this change?

**Comment by jpdriver** [comment]
@stevenocchipinti this was published with `3.0.0` earlier today

see https://github.com/FormidableLabs/react-live/releases/tag/v3.0.0 for release notes, and you can install using

`yarn add react-live@next`

**Comment by stevenocchipinti** [comment]
Awesome! Thank you!
